lib/logger/lib/logger/watcher.ex

```diff
@@ -49,8 +52,12 @@ defmodule Logger.Watcher do
   end
 
   def handle_info({:gen_event_EXIT, handler, reason}, {mod, handler} = state) do
-    _ = Logger.error ":gen_event handler #{inspect handler} installed at #{inspect mod}\n" <>
-                 "** (exit) #{format_exit(reason)}"
+    _ =
+      Logger.error(
+        ":gen_event handler #{inspect(handler)} installed at #{inspect(mod)}\n" <>
+          "** (exit) #{format_exit(reason)}"
+      )
```

Comment:
We can pass iodata to Logger.

```elixir
message = [
  ":gen_event handler ",
  inspect(handler),
  " installed at ",
  inspect(mod),
  ?\n,
  "** (exit) ",
  format_exit(reason),
]

_ = Logger.error(message)
```

@josevalim thoughts?

Reply:
Yeah, I like it.